lifecycle-operator/chart/values.yaml

```diff
@@ -174,6 +174,12 @@ podAnnotations: {}
 schedulingGatesEnabled: false
 ## @param  allowedNamespaces specifies the allowed namespaces for the lifecycle orchestration functionality
 allowedNamespaces: []
+## @param  deniedNamespaces specifies a list of namespaces where the lifecycle orchestration functionality is disabled, ignored if `allowedNamespaces`` is set
```

Comment:
```suggestion
## @param  deniedNamespaces specifies a list of namespaces where the lifecycle orchestration functionality is disabled, ignored if `allowedNamespaces` is set
```

Reply:
Good catch! Resolved and adjusted in `README` as well.